src/test/java/sorinfoca/Tests/Tema8/ConfigPage/ConfigPages.java

```diff
@@ -0,0 +1,37 @@
+package sorinfoca.Tests.Tema8.ConfigPage;
+
+import org.openqa.selenium.chrome.ChromeDriver;
+import org.openqa.selenium.chrome.ChromeOptions;
+import org.openqa.selenium.support.ui.WebDriverWait;
+import org.testng.annotations.AfterTest;
+import org.testng.annotations.BeforeTest;
+
+import java.time.Duration;
+import java.util.concurrent.TimeUnit;
+
+public class ConfigPages {
+
+    private WebDriverWait wait;
+
+    protected ChromeDriver driver = null;
+
+    private static String baseUrl = "https://testpages.herokuapp.com/";
+
+    public static String getBaseUrl() {
+        return baseUrl;
+    }
+
+    @BeforeTest
+    public ChromeOptions setUp() {
```

Comment:
This method shouldn't return anything so switch to void type. You can also rename it to desktopSetup()

Reply:
Also consider to add onlyForGroups attribute for desktop, something like ->
![image](https://user-images.githubusercontent.com/18215854/213879561-f2c8a2cc-d3e4-49fc-83b5-d4185bafa5e0.png)